std/owned.sol

```diff
@@ -9,7 +9,7 @@ contract owned {
         }
     }
 
-    function owned() public {
+    constructor() public {
```

Comment:
I wouldn't change this file yet, especially as the version pragma on the top wasn't updated. This file can be used by contracts referring to the github repo.